### ISSUE TITLE
Baseline with a test of just the React tree

### ISSUE THREAD

**Issue by jamiebuilds** [modulz/stitches-bench#3]
The benchmarks don't tell you much about what overhead stitches is adding, only that it's less than other libraries

**Comment by StephenHaney** [comment]
Hey Jamie, good shout... at the moment this first version of our benches assume the user has chosen a CSS-in-JS strategy as that's the case for the majority of users.

We can add a comparison to plain CSS to make the trade-off information available for those who are interested. What would you be most interested in seeing, measure mounting the same React tree but just use CSS classes to achieve the same styling?

**Comment by jamiebuilds** [comment]
Yeah, I would use a combination of `className` with a stylesheet and `style` with any dynamic styles. You could also make use of CSS vars to do stuff like:

```css
.box:active {
  color: var(--box-active-color);
}
```

```jsx
<div className="box" style={{ "--box-active-color": "dodgerblue" }}/>
```

For those dynamic styles you can't put directly in `style` (i.e. `:hover`, `:active`, etc)
